core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -83,14 +83,16 @@ class DataFileTestCase(LogBufferTestCase):
     particular validator class and collect the log records emitted.
     """
 
-    def validate(self, data_filename, validator_class, extra_meta_fields=None):
+    def validate(self, data_filename, validator_class, extra_meta_fields=None, relaxed_mode=None):
         """Validate a file with a Validator and return the log records."""
         meta_dict = {'data_filename': data_filename}
         if extra_meta_fields is not None:
             meta_dict.update(extra_meta_fields)
+        if relaxed_mode is None:
+            relaxed_mode = False
```

Comment:
Just a note: the reason that the default for `extra_meta_fields` is `None` instead of the dictionary that will actually be used is that dictionaries are mutable in place, and using mutable objects as default arguments in Python can lead to [confusing situations](http://effbot.org/zone/default-values.htm) if not specifically intended. There is no real need not to use `False` as a default, booleans are not mutable objects.